### ISSUE TITLE
Lot size option of fixed volume

### ISSUE THREAD

**Issue by venomdev** [wait4signal/sharing-is-caring#11]
I would like to add a fixed lot size volume so that the receiver will use the same volume regardless of what the provider volume is using.

I am happy to write the code myself if you want to add it to the next release.

**Comment by bgabela** [comment]
Hi @venomdev , that is not something I needed for my use-case as I prefer to scale proportionally. I don't have capacity to add it but happy to accept your changes.
Potential issue with such a setup is that it could skew risk management so the exposure/drawdown would now be different between provider and consumer. Not a big deal but something to keep in mind.

**Comment by venomdev** [comment]
Maybe I can add this feature in the same update, they would be similar fixes.

https://github.com/wait4signal/sharing-is-caring/issues/1#issue-1089341343

**Comment by bgabela** [comment]
> Maybe I can add this feature in the same update, they would be similar fixes.
> 
> [#1 (comment)](https://github.com/wait4signal/sharing-is-caring/issues/1#issue-1089341343)

Yeah they are similar, would be great if you can help with flexible lot multiplier as requested in that issue

**Comment by venomdev** [comment]
I seem to have a working version so I'll run some tests with it for a few days and upload the feature soon.
